be/src/runtime/vectorized/chunk_cursor.h

```diff
@@ -40,18 +54,26 @@ class ChunkCursor {
 
     [[nodiscard]] ChunkPtr clone_empty_chunk(size_t reserved_row_number) const;
 
+    Status chunk_supplier(Chunk**);
+    bool chunk_probe_supplier(Chunk**);
+    bool chunk_has_supplier();
+
 private:
     void _reset_with_next_chunk();
 
 private:
     ChunkSupplier _chunk_supplier;
+    ChunkProbeSupplier _chunk_probe_supplier;
```

Comment:
Why in sort context，we have the `probe` operation？

Reply:
> Why in sort context，we have the `probe` operation？

Probe for chunks, because _chunk_queue maybe empty.

Reply:
I means In sort context，we don't have `probe` semantics.

Reply:
> I means In sort context，we don't have `probe` semantics.

So what's the name here?